cmd/core/pdr_creation_context.go

```diff
@@ -78,6 +79,13 @@ func (pcc *PDRCreationContext) extractPDR(pdr *ie.IE, spdrInfo *SPDRInfo) error
 		}
 		return fmt.Errorf("F-TEID IE is missing")
 	} else if ueIP, err := pdr.UEIPAddress(); err == nil {
+		if ueIP.Flags&(1<<2) != 0 {
+			if ip, err := pcc.getIP(pcc.Session.RemoteSEID); err == nil {
```

Comment:
Do we rally need to pass a parameter to `getIP`? Seems like pcc can access it directly within method